Exception: STATUS_ACCESS_VIOLATION at rip=001004869C7
rax=0000000000000000 rbx=00000000FFFFC680 rcx=00000000FFFFC680
rdx=FFFFFFFFFFFFFFE0 rsi=0000000000000000 rdi=FFFFFFFFFFFFFFE0
r8 =00000006000747F0 r9 =0000000000000000 r10=0000000100000000
r11=0000000100426B89 r12=00000000FFFFC6E0 r13=0000000000000000
r14=0000000000000002 r15=0000000000000000
rbp=00000000FFFFC6A0 rsp=00000000FFFFC5F0
program=D:\userdata\annoviko\workspace\pyclustering\ccore\utcore\utcore.exe, pid 9312, thread main
cs=0033 ds=002B es=002B fs=0053 gs=002B ss=002B
Stack trace:
Frame        Function    Args
000FFFFC6A0  001004869C7 (00600074720, 00000000000, 00100482F36, 00000000010)
000FFFFC6A0  00100427888 (000FFFFC828, 0010044E240, 00180161C0B, 001004940C0)
00600034090  0010042CFCD (00000000000, 00600033C50, 0010047915B, 001801F6B30)
00100497BC3  0010047E702 (00180183FEC, 00600033CF0, 001801565C8, 00000000021)
000FFFFC940  001004651C5 (0018015657B, 1D18C3A6149FD4F, 00100033270, 00000000011)
00600023220  00100465555 (00100000002, 00000000000, 001801F6B30, 000FFFFD680)
153D2D47B2C  001004657A5 (00100000001, 000FFFFCB90, 000FFFFCC60, 00600023428)
003DD17EC00  00100466977 (000FFFFCCC0, 00000000000, 000FFFFCC60, 00180337F78)
000FFFFCCC0  00100466D94 (00000000000, 00000000000, 00000000000, 000FFFFCC60)
000FFFFCCC0  0010048951B (00000000020, 30001000000FF00, 0018004843C, 000FFFFD680)
000FFFFCCC0  001800484AD (00000000000, 00000000000, 00000000000, 00000000000)
00000000000  0018004618C (00000000000, 00000000000, 00000000000, 00000000000)
000FFFFFFF0  00180046224 (00000000000, 00000000000, 00000000000, 00000000000)
End of stack trace
